packages/react-core/src/components/NotificationDrawer/__tests__/Generated/__snapshots__/NotificationDrawer.test.tsx.snap

```diff
@@ -1,11 +1,12 @@
 // Jest Snapshot v1, https://goo.gl/fbAQLP
 
 exports[`NotificationDrawer should match snapshot (auto-generated) 1`] = `
-<div
-  className="pf-c-notification-drawer ''"
+<NotificationDrawerBase
+  className="''"
+  innerRef={null}
 >
   <div>
     ReactNode
   </div>
-</div>
+</NotificationDrawerBase>
```

Comment:
I'm unsure if the test here is still really testing the things that need to be tested based on the snapshot.